okapi-core/src/main/raml/okapi.raml

```diff
@@ -962,7 +1022,8 @@ traits:
               text/plain:
       post:
         is: [
-          versionFilters
+          versionFilters,
+          installParameters
         ]
         description: Enable, disable or upgrade one or more modules for
           tenant. The request body and response body is of the same type
```

Comment:
Probably need a period at the end of the sentence here.

Reply:
It's there on next line.. But I can move it up.